src/selectors/Entities/vaccinePrescription.js

```diff
@@ -99,3 +99,26 @@ export const selectSelectedSupplementalData = state => {
 
   return supplementalData;
 };
+
+export const selectSupplementalDataIsValid = state => {
+  const VaccinePrescriptionState = selectSpecificEntityState(state, 'vaccinePrescription');
+  const { isSupplementalDataValid } = VaccinePrescriptionState;
+
+  return isSupplementalDataValid;
+};
+
+export const selectLastSupplementalData = () => {
+  const inQuery = UIDatabase.objects('Transaction')
+    .filtered("type == 'customer_invoice' AND customData != null")
+    .map(({ id }) => `transaction.id == "${id}"`)
+    .join(' OR ');
+  const fullQuery = `(${inQuery}) AND itemBatch.item.isVaccine == true`;
+  const result = inQuery
+    ? UIDatabase.objects('TransactionBatch')
+        .filtered(fullQuery)
+        .sorted('transaction.entryDate', true)
+        .slice()
```

Comment:
nit: getting this linter error 🤷 

![image](https://user-images.githubusercontent.com/35858975/125005889-fbe6b980-e0b0-11eb-9c43-ae7ee7ac0a42.png)

Reply:
Ohh yeah, I tried removing those and got _another_ warning, other than disabling for the whole file, any ideas? 🤣 :
![image](https://user-images.githubusercontent.com/65875762/125009162-1708f780-e0b8-11eb-95fd-d880f0cdd70c.png)

Reply:
nah .. sorry .. I usually just steal a linter config from somewhere and run with it .. never really looked to deeply into them... uhh.. I think in the `eslintrc` folder maybe just making `"indent": ["warning", 2, { "SwitchCase": 1 }],`  🤣